doc/docusaurus/docs/using-plutus-tx/running-debugging.md

```diff
@@ -0,0 +1,301 @@
+---
+sidebar_position: X
+---
+
+# Running and Debugging Compiled Code
+
+You can run compiled code locally (without the need of a Cardano Node)
```

Comment:
I think the for most people, the most useful functionality of the executable is optimizing UPLC. This should be made clear at the beginning. Debugging, on the other hand, should be de-emphasized, since we don't really have a working debugger.

The initial examples should also focus on how to use it for UPLC optimization, before talking about anything else.